RELEASES.md

```diff
@@ -2,6 +2,22 @@
 
 --------------------------------------------------------------------------------
 
+## 0.33.0
+
+Released 2022-01-05.
+
+### Added
+
+* Compiled wasm modules may now omit information about mapping addresses to
+  source locations, resulting in smaller binaries but a worse debugging
+  experience.
```

Comment:
The wording here reads a little ambiguously... "added: worse debugging experience". Maybe we could say "may not optionally omit information ... if configured by a setting"?